cfme/storage/volume.py

```diff
@@ -84,6 +87,7 @@ def in_volume(self):
 class VolumeAllView(VolumeView):
     toolbar = View.nested(VolumeToolbar)
     search = View.nested(Search)
+    paginator = PaginationPane()
```

Comment:
Required:
No need of `paginator` as we are using `BaseEntitiesView`. you can easy access it like
```
view.entities.paginator
```